endpoints/openrtb2/auction.go

```diff
@@ -262,8 +262,13 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request) (req *openrtb_
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
+	impInfo, errs := parseImpInfo(requestJson)
+	if len(errs) > 0 {
+		return nil, nil, errs
+	}
```

Comment:
The error scenarios seem to not be covered. Can we add test scenarios that throw error?
![image](https://user-images.githubusercontent.com/15863840/130526839-e859be72-e709-4485-af32-f4eb8c4b5012.png)

Reply:
Added